File: Data Structures & Algorithms/top-k-elements-in-list/submission-8.java
class Solution {
    public int[] topKFrequent(int[] nums, int k) {
        Map<Integer, Integer> count = new HashMap<>();
        for(int num: nums){
            count.put(num, count.getOrDefault(num, 0) + 1);
        }
        List<Map.Entry<Integer,Integer>> entries = new ArrayList<>(count.entrySet());
        entries.sort((a,b)-> Integer.compare(b.getValue(),a.getValue()));

        int[] answer = new int[k];
        for(int i = 0; i < k; i++){
            answer[i] = (entries.get(i).getKey());
        }
        return answer;
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-8.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        answer = [0,0]
        for i in range(len(nums) - 1,-1,-1):
            if (target - nums[i]) in nums :
                answer[1] = nums[i]
                answer[0] = target - nums[i]
                break
        print(answer)
        answer[0] = nums.index(answer[0])
        for i in range(answer[0] + 1, len(nums)):
            if nums[i] == answer[1]:
                print("true")
                answer[1] = i
                break
        
        return answer